atomicwrites/__init__.py

```diff
@@ -146,7 +148,7 @@ def _open(self, get_fileobject):
         f = None  # make sure f exists even if get_fileobject() fails
         try:
             success = False
-            with get_fileobject() as f:
+            with get_fileobject(**self._tempfile_kwargs) as f:
```

Comment:
Could you rename this param to open_kwargs

Reply:
Quite right. Meant to do that before. Fixed!